lib/telegraf/telegraf.conf.erb

```diff
@@ -634,6 +689,13 @@
 #   ##
 #   # interfaces = ["eth0"]
 
+  ## These lines fix the system metric names to basically be
+  ## system.<host>.<measurement>.<field>
+  ## Look at the output template for more info.
+  name_override = "system"
+  [inputs.net.tags]
+    system_measurement_tag = "net"
```

Comment:
Is it worth expanding this to "network" to match the mostly other unabbreviated ones?